src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotConstantPool.java

```diff
@@ -435,6 +435,7 @@ private int getNameRefIndexAt(int index) {
      * another entry denoted by {@code which}.
      *
      * @param which constant pool index or constant pool cache index
+     * @param opcode bytecode
```

Comment:
Is this a param?  You should remove the jvmci changes because they're not needed for this change.

Reply:
Or should the comment say that 'which' is the constant pool index only in this case?

Reply:
The JVMCI should still be updated to match the internal functionality.